test/tizen_capi/unittest_tizen_capi_layer.cpp

```diff
@@ -77,7 +77,7 @@ TEST(nntrainer_capi_nnlayer, setproperty_01_p) {
 }
 
 /**
- * @brief Neural Network Layer Create / Delete Test (negative test )
+ * @brief Neural Network Layer Set Property Test (positive test )
```

Comment:
BTW, is there any _special_ reason to add an white space between `test` and `)`?

Reply:
nop. It's typo. Thanks.